### PR TITLE
Added Floating Button

### PR DESCRIPTION
**Pull request by priyanshk20**
floatingButton

### CODE REVIEW

**Review by vinayak-trivedi — COMMENTED** [review]
This PR contains all the files that were changed in your previous PR. I don't think that should be the case

File: src/components/FloatingButton.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    },
+    {
+      text: "To Do",
+      icon: require("../../assets/task.201x256.png"),
+      buttonSize:50,
+      name: "bt_todo",
+      position: 3,
+      color : '#492ed1' 
+    },
+  ];
```
Comment (vinayak-trivedi) [inline src/components/FloatingButton.tsx]: Can we move the actions in a different file to make it a bit cleaner

Comment (priyanshk20) [reply src/components/FloatingButton.tsx]: Sure

Comment (priyanshk20) [reply src/components/FloatingButton.tsx]: Done

**Review by dakshbhardwaj — DISMISSED** [review]
LGTM

File: src/components/CreateGoalForm/Dropdown.tsx [hunk 2]
```diff
@@ -17,7 +17,7 @@ const local_data = [
 
 export interface Props {}
 
-const SelectCountryScreen: React.FC<Props> = () => {
+const SelectCountryScreen: React.FC<Props> = _props => {
   const [duration, setDuration] = useState('1');
```
Comment (vinayak-trivedi) [inline src/components/CreateGoalForm/Dropdown.tsx]:
```suggestion
  const [duration, setDuration] = useState<string>('1');
```

File: src/components/CreateGoalForm/Dropdown.tsx [hunk 3]
```diff
@@ -32,26 +32,39 @@ const SelectCountryScreen: React.FC<Props> = () => {
...
+
         <View>
+
           <Text style={styles.formchild}>Roadmap Type</Text>
           <Radio />
 
           <Text style={styles.formchild}>Existing Roadmap</Text>
           <Searchbar />
         </View>
+        </View>
```
Comment (vinayak-trivedi) [inline src/components/CreateGoalForm/Dropdown.tsx]: please fix the indentation here.

File: src/components/CreateGoalForm/Dropdown.tsx [hunk 4]
```diff
@@ -32,26 +32,39 @@ const SelectCountryScreen: React.FC<Props> = () => {
...
+
           <Text style={styles.formchild}>Associated Long Term Goal</Text>
           <Searchbar />
         </View>
-      )}
-    </View>
+        </View>
+      )
+      }
+      </View>
```
Comment (vinayak-trivedi) [inline src/components/CreateGoalForm/Dropdown.tsx]: same as above, fix the indentation here

**Review by vinayak-trivedi — DISMISSED** [review]

File: src/components/CreateGoalForm/Dropdown.tsx (hunk 2 above)
Comment (priyanshk20) [reply src/components/CreateGoalForm/Dropdown.tsx]: done

File: src/components/CreateGoalForm/Dropdown.tsx (hunk 3 above)
Comment (priyanshk20) [reply src/components/CreateGoalForm/Dropdown.tsx]: done

File: src/components/CreateGoalForm/Dropdown.tsx (hunk 4 above)
Comment (priyanshk20) [reply src/components/CreateGoalForm/Dropdown.tsx]: done

**Comment by shreya-mishra** [comment]
> This PR contains all the files that were changed in your previous PR. I don't think that should be the case

Done that

**Review by shreya-mishra — DISMISSED** [review]
LGTM

**Review by shreya-mishra — APPROVED** [review]

**Pull request merged**